src/main/webapp/WEB-INF/tags/instructor/feedbackEdit/newQuestionForm.tag

```diff
@@ -137,8 +137,11 @@
   <input type="hidden" name="<%= Const.ParamsNames.FEEDBACK_QUESTION_SHOWGIVERTO %>" >
   <input type="hidden" name="<%= Const.ParamsNames.FEEDBACK_QUESTION_SHOWRECIPIENTTO %>" >
   <input type="hidden" name="<%= Const.ParamsNames.USER_ID %>" value="${data.account.googleId}">
-  <input type="hidden" name="<%= Const.ParamsNames.FEEDBACK_QUESTION_GENERATEDOPTIONS %>"
+  <input type="hidden" name="<%= Const.ParamsNames.FEEDBACK_QUESTION_MCQ_GENERATED_OPTIONS %>"
+         value="<%= FeedbackParticipantType.NONE.toString() %>"
+         id="<%= Const.ParamsNames.FEEDBACK_QUESTION_MCQ_GENERATED_OPTIONS %>">
```

Comment:
Why we need `id` here? This could introduce possibility of duplicated id in the same HTML file.

Reply:
I don't understand how the duplicate id might happen?
Initially, there was `id` field which is being used in tests to get the element.

Reply:
I feel a little bit strange here since there is not suffix for the id. (Yeah, I know, it's new question from, there won't be another new question form in the same page :P)

Further investigation tells me that this might be legacy code.
![image](https://user-images.githubusercontent.com/13977235/37460911-a88f3ee0-2887-11e8-8ac1-c61a63d70764.png)
You can see we actually have two `name="generatedOptions"` here. I suspect we always get the first one and therefore there is no bug currently.

Reply:
I can understand the part of not having the suffix for id, but the `mcqGeneratedOptions` id is actually used for sending the details of the question and as you said only one new question can be made at a time so it works as expected.

The thing is I need to have the `id` for the hidden input of the field as its used a lot of times in test to check for the value.

So should I go and put back the `id` for the hidden input of mcq and msq as before?

Reply:
@xpdavid waiting for your input on this https://github.com/TEAMMATES/teammates/pull/8584#discussion_r175252240

Reply:
Ok. I see some misunderstanding here. When u get parameters from the server side, u actually refer to the `name` attribute rather than `id` attribute.

Refer to my previous comment:
>>> I suspect we always get the first one and therefore there is no bug currently.

It looks like we could delete this line without break anything. I hope u can understand my concern here as we don't want to just make things work without knowing why it works.

Reply:
Thanks. I understand better now.
Looking into it